src/AppBundle/Controller/DangerPointController.php

```diff
@@ -72,6 +72,12 @@ public function getAllAction()
     {
         if ($this->get('security.authorization_checker')->isGranted('ROLE_ADMIN')) {
             $restresult = $this->getDoctrine()->getRepository('AppBundle:DangerPoint')->findAll();
+        } elseif (
```

Comment:
The swagger documentation for all functions in this file must include the 404 responses. They aren't all documented.

Reply:
this was not part of the changed code, but I fixed it